draft-ietf-quic-recovery.md

```diff
@@ -410,8 +410,9 @@ time threshold MUST be set to at least kGranularity.  The time threshold is:
 max(kTimeThreshold * max(smoothed_rtt, latest_rtt), kGranularity)
 ~~~
 
-If packets sent prior to the largest acknowledged packet cannot yet be declared
-lost, then a timer SHOULD be set for the remaining time.
+When an ACK frame is received and unacknowledged packets sent prior to the
```

Comment:
This is slightly less correct, because a loss detection timer could be set for one packet, and then it could be declared lost, and then when it is, the timer may be re-armed for the next packet to be declared lost.